.github/workflows/health.yaml

```diff
@@ -4,19 +4,21 @@ on:
     branches: [ main ]
     # Remove paths after https://github.com/bmw-tech/dart_apitool/issues/177 is addressed.
     paths:
-      - "pkgs/ffi/**"
-      - "pkgs/native_assets_builder/**"
-      - "pkgs/native_assets_cli/**"
-      - "pkgs/native_toolchain_c/**"
+      - ".github/workflows/health.yaml"
+      - "pkgs/**"
     types: [opened, synchronize, reopened, labeled, unlabeled]
 jobs:
   health:
     uses: dart-lang/ecosystem/.github/workflows/health.yaml@main
     with:
       coverage_web: false
       # TODO(https://github.com/dart-lang/native/issues/1242): Add coverage back.
-      checks: "version,changelog,license,do-not-submit,breaking,leaking"
-      use-flutter: true
-      sdk: master
+      checks: "changelog,license,do-not-submit,breaking,leaking"
+      flutter_packages: "pkgs/ffigen,pkgs/jni,pkgs/jnigen,pkgs/objective_c"
+      ignore_license: "**.g.dart"
+      ignore_coverage: "**.mock.dart,**.g.dart"
+      ignore_packages: "pkgs/swiftgen,pkgs/jnigen,pkgs/swift2objc"
+      sdk: dev
+      channel: master
```

Comment:
Would it make sense to also run it on the other channels? E.g. if you stack PRs you merge the one PR into the next PR, then the health checks don't run currently.